docs/release-notes/v11.7.0.md

```diff
@@ -0,0 +1,165 @@
+---
+id: v11.7.0
+title: Version 11.7.0 Release Notes
+sidebar_label: v11.7.0
+image: "img/release-note-link-preview.png"
+---
+
+_Oct 12th, 2023_
+
+## Console
+
+### New Features
+
+#### User feedback in Console
+
+From now on, you can suggest ideas, report issues, and communicate any other feedback to help us improve the Product experience directly in Console, without the need to change your webpage.  
+You can leave us precious feedback by clicking on the related Feedback icon, which is available from the Console Launchbar. 
+
+All the collected feedback will be addressed to the Console managers, who will manage and process it within a brand new section of the CMS.
+
+For more details, [go to the official documentation](./development_suite/user-feedback/give-feedback.md).
+
+![User feedback modal](./img/v11.7.0/user-feedback.png)
+
+#### New runtime commands for Mia-Platform Miactl CLI 
+
+With this release, new runtime commands are now available on Miactl CLI.
+
+More specifically, the following commands have been introduced on the command line:
+
+* Visualization of all Environments of a Console Project;
+* List of Pods/Cronjobs/Jobs of an Environment of a Console Project;
+* Visualization of all the Events of a specific resource in the current workload of an Environments of a Console Project;
+* Manually launch a CronJob as a Job.
+
+### Improvements
+
+### Bug Fix
+
+This version addressed the following bugs concerning the creation of a new project:
+
+* It has been fixed a bug that prevented the creation of a project in case the user had returned from step 2 (or subsequent) to step 1 to change some data;
+* A problem on the length of the description of a new project that caused its creation to fail has been corrected;
```

Comment:
```suggestion
This version addressed the following bugs concerning the creation of a new Project:

* It has been fixed a bug that prevented the creation of a Project in case the user had returned from step 2 (or subsequent) to step 1 to change some data;
* A problem on the length of the description of a new Project that caused its creation to fail has been corrected;
```